src/IssueViz.Security/Taint/TaintIssuesSynchronizer.cs

```diff
@@ -123,9 +131,17 @@ public async Task SynchronizeWithServer()
             }
         }
 
+        private async Task<AnalysisInformation> GetAnalysisInformation(string projectKey)
+        {
+            var branches = await sonarQubeService.GetProjectBranchesAsync(projectKey, CancellationToken.None);
+            var issuesBranch = branches.First(x => x.IsMain);
```

Comment:
How defensive do we need to be about bugs on the SQ/SC side?
If we throw an exception here, will it bring down VS, or just give a gold bar?

Reply:
No need; the entire block is still covered with a try-catch, so we should be ok (I threw an exception and it was fine).

Also, if the user opens the window in an unbound project, we won't reach this code at all (since there are no issues).
So we could fail only if there _are_ issues, but for some reason there is no main branch. In which case, we will get to the catch block and log the exception.